cpp/src/Ice/SSL/SecureTransportEngine.cpp

```diff
@@ -619,76 +618,35 @@ SecureTransport::SSLEngine::initialize()
 
     if (!certFile.empty())
     {
-        vector<string> files;
-        if (!IceInternal::splitString(certFile, IceInternal::pathsep, files) || files.size() > 2)
+        string resolved;
+
+        if (!checkPath(certFile, defaultDir, false, resolved))
```

Comment:
Might be nicer if `checkPath` returned an `std::optional<std::string>>` instead of passing in `resolved` and checking a bool